daemon/mlops-agent-internal.c

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * @file    mlops-agent-internal.c
+ * @date    20 January 2025
+ * @brief   Internal function for ml-agent interface.
+ * @see     https://github.com/nnstreamer/deviceMLOps.MLAgent
+ * @author  Jaeyun Jung <jy1210.jung@samsung.com>
+ * @bug     No known bugs except for NYI items
+ */
+
+#include "log.h"
+#include "mlops-agent-internal.h"
+#include "mlops-agent-node.h"
+#include "service-db-util.h"
+
+/**
+ * @brief Internal function to initialize mlops-agent interface.
+ */
+void
+ml_agent_initialize (const char *db_path)
+{
+  svcdb_initialize (db_path);
```

Comment:
It would be nice to check null before using ```db_path```.

Reply:
Thanks, I added assertion to prevent invalid function call.